internal/helper/request_url.go

```diff
@@ -0,0 +1,39 @@
+package helper
```

Comment:
Done. For adding this to the main branch, can we add unused/dead code to the main branch? Wouldn't it be better to replicate this helper in case you need it.

Reply:
cc: @shreyassreenivas

Reply:
PTAL PR #51

Reply:
Please rebase

Reply:
Done @Vasundharashukla